calabash-cucumber/lib/calabash-cucumber/logging.rb

```diff
@@ -0,0 +1,79 @@
+module Calabash
+  module Cucumber
+    require "run_loop"
+
+    # These methods are not part of the API.
+    #
+    # They may change at any time.
+
+    # !@visibility private
+    # blue
+    def self.log_warn(msg)
+      puts self.blue(" WARN: #{msg}") if msg
+    end
+
+    # !@visibility private
+    # magenta
+    def self.log_debug(msg)
+      if RunLoop::Environment.debug?
+        puts self.magenta("DEBUG: #{msg}") if msg
+      end
+    end
+
+    # !@visibility private
+    # green
+    def self.log_info(msg)
+      puts self.green(" INFO: #{msg}") if msg
+    end
+
+    # !@visibility private
+    # red
+    def self.log_error(msg)
+      puts self.red("ERROR: #{msg}") if msg
+    end
+
+    private
+
+    # @!visibility private
+    def self.windows_env?
+      RbConfig::CONFIG["host_os"] =~ /mswin|mingw|cygwin/
+    end
+
+    # @!visibility private
+    def self.colorize(string, color)
```

Comment:
Teehee... I've seen this code somewhere. 

I believe there is a `colorize` gem for ruby, no? It's pretty easy to use, like 
```ruby
"my_string".green
"my_string".blue.on_white
```